test/api_save_product_test.dart

```diff
@@ -398,6 +398,82 @@ Like that:
       }
     }, skip: 'Works randomly');
 
+    test('Test saving food with PerSize.serving Nutrients', () async {
```

Comment:
Instead of copying/pasting another test, I guess you could loop on `PerSize.values`.

Reply:
I went ahead and combined the test with the test I copied from